pkg/cmd/certmanager_app.go

```diff
@@ -71,7 +67,7 @@ func makeInstallCertManager() *cobra.Command {
 			}
 		}
 
-		nsRes, nsErr := kubectlTask("create", "namespace", namespace)
+		nsRes, nsErr := kubectl(kubeConfigPath, "", "create", "namespace", namespace).Execute()
```

Comment:
What's the empty parameter for?

Reply:
Initially I had the idea to also add kube context as a parameter and wanted to have it prepared for it.

The new function signature of kubectl(kubeConfig, kubeContext, ...args) where kubeContext is not yet used.